pkg/bindings/util/util.go

```diff
@@ -11,14 +12,25 @@ import (
 )
 
 func IsSimpleType(f reflect.Value) bool {
+	switch f.Interface().(type) {
+	case fmt.Stringer:
+		return true
+	}
+
 	switch f.Kind() {
 	case reflect.Bool, reflect.Int, reflect.Int64, reflect.Uint, reflect.Uint64, reflect.String:
 		return true
 	}
+
 	return false
 }
 
 func SimpleTypeToParam(f reflect.Value) string {
+	switch cast := f.Interface().(type) {
```

Comment:
nit: probably could be:
```go
if s, ok := f.Interface().(Stringer); ok {
    return s.String()
}

Reply:
Changed in https://github.com/containers/podman/pull/9414/commits/e022c19753182cfb85f8f49354d493ee3a3147a3.